src/test/java/com/roommatematcher/enterprise/RoommateMatcherApplicationTests.java

```diff
@@ -356,4 +357,19 @@ private void thenReturnOfLocationNumberOfResidents5() {
         assertEquals(5,numberOfResidents);
     }
 
+    @Test
+    void fetchAllProfiles_returnListOfProfiles(){
+        givenThereAre7Profiles();
+        whenFetchAllIsCalled();
+        returnListOf7Profiles();
+    }
+
+    private void givenThereAre7Profiles() {
+    }
+
+    private void whenFetchAllIsCalled(){
+    }
+
+    private void returnListOf7Profiles(){
+    }
```

Comment:
The @Test annotation is a bit misleading, as there are no asserts here.

This appears to be an incomplete test.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.